pkg/kubelet/apis/deviceplugin/v1beta1/constants.go

```diff
@@ -31,3 +31,5 @@ const (
 	// KubeletSocket is the path of the Kubelet registry socket
 	KubeletSocket = DevicePluginPath + "kubelet.sock"
 )
+
+var SupportedVersions = [...]string{"v1alpha2", "v1alpha", "v1beta1"}
```

Comment:
I think during 1.10 time frame, we should drop the list to v1beta1 only, unless we plan to implement multiple versions of endpoint to support alpha versions.